src/main/groovy/org/labkey/gradle/plugin/FileModule.groovy

```diff
@@ -510,7 +510,7 @@ class FileModule implements Plugin<Project>
     private static void addDependencies(Project project)
     {
         Project serverProject = BuildUtils.getServerProject(project)
-        if (serverProject != null)
+        if (serverProject != null && serverProject != project)
```

Comment:
This implies that the FileModule plugin would get applied to the `serverProject`, which doesn't seem like it should ever happen.  Does it?

Reply:
It does. The 'ModuleDistribution' task depends on the 'stageApp' task in the server project. The standalone module defines `gradle.ext.serverProjectPath=":"`in its `parameters.gradle` and the 'standalone' plugin applies the 'serverDeploy' plugin to define that task.
`serverProject != project` here prevents a circular dependency in that case.